templates/snippets/header.html

```diff
@@ -1,7 +1,8 @@
 {% comment %}
 <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 bg-dark border-bottom shadow-sm sticky-top">
-    <h2 class="mb-3 my-md-0 mr-md-auto font-weight-normal" >
-        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>								<!--Change this-->
+    <h2 class="mb-3 my-md-0 mr-md-auto font-weight-normal">
+        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>
+        <!--Change this-->
```

Comment:
Why have you added this comment?

Reply:
Actually I have an extension in VSCode which automatically beautifies the code. Should I make it in the way as it was before? I have not added anything, only the formatting is changed

Reply:
The new formatting is fine, just remove the `<!--Change this-->` comment.

What's the extension btw?

Reply:
Simple React Snippets